platform/ui/src/components/ToolbarButton/ToolbarButton.tsx

```diff
@@ -73,15 +50,11 @@ const ToolbarButton = ({
 
 ToolbarButton.defaultProps = {
   dropdownContent: null,
-  isActive: false,
-  type: 'action',
 };
 
 ToolbarButton.propTypes = {
   /* Influences background/hover styling */
-  type: PropTypes.oneOf(['action', 'toggle', 'tool']),
   id: PropTypes.string.isRequired,
-  isActive: PropTypes.bool,
   className: PropTypes.string,
   commands: PropTypes.arrayOf(
```

Comment:
Can you update this to take the simple type or the full array, and also use the Types value exported from the commands?  That way it just becomes commands: PropTypes.instanceOf(Types.Commands)  (something like that anyways).

Reply:
This one is tricky since it is in UI, not sure the best way to do this really